selfdrive/car/gm/carstate.py

```diff
@@ -51,7 +51,11 @@ def update(self, pt_cp, cam_cp, loopback_cp):
     # To avoid a cruise fault we need to match the ECM's brake pressed signal and threshold
     # https://static.nhtsa.gov/odi/tsbs/2017/MC-10137629-9999.pdf
     ret.brake = pt_cp.vl["ECMAcceleratorPos"]["BrakePedalPos"]
-    ret.brakePressed = ret.brake >= 8
+    if self.CP.networkLocation != NetworkLocation.fwdCamera:
+      ret.brakePressed = ret.brake >= 8
+    else:
+      # Match the ECM's brake pressed threshold at a stop
+      ret.brakePressed = ret.brake >= 20
```

Comment:
can you add a sentence or two to the comment here explaining why this threshold is necessary?